swingset/src/main/java/com/nqadmin/swingset/utils/SSComponentInterface.java

```diff
@@ -139,6 +128,13 @@ default void bind(final RowSet _rowSet, final String _boundColumnName) {// throw
 
 	}
 
+	/**
+	 * Override this method for notification of a change in metadata.
+	 * Typically from setRowSet() or bind().
+	 */
+	default void metadataChange() {
+	}
+
```

Comment:
Formalize notification of metadata change. Seems better than inferring that there's a change.